services/src/main/scala/cromwell/pipeline/service/GitLabProjectVersioning.scala

```diff
@@ -47,7 +48,26 @@ class GitLabProjectVersioning(httpClient: HttpClient, config: GitLabConfig)
   override def getFileTree(project: Project, version: Option[Version])(
     implicit ec: ExecutionContext
   ): AsyncResult[List[String]] = ???
+
   override def getFile(project: Project, path: Path, version: Option[Version])(
     implicit ec: ExecutionContext
-  ): AsyncResult[String] = ???
+  ): AsyncResult[ProjectFile] = {
+    val filePath: String = URLEncoder.encode(path.toString, "UTF-8")
+    val fileVersion: String = config.defaultFileVersion
```

Comment:
You forgot to use `version` from method parameters.

Reply:
Done, thanks!